runtime-linux.md

```diff
@@ -1,11 +1,5 @@
 # Linux Runtime
 
-## File descriptors
-
-By default, only the `stdin`, `stdout` and `stderr` file descriptors are kept open for the application by the runtime.
-The runtime MAY pass additional file descriptors to the application to support features such as [socket activation](http://0pointer.de/blog/projects/socket-activated-containers.html).
-Some of the file descriptors MAY be redirected to `/dev/null` even though they are open.
```

Comment:
While the first sentence is misleading, we shouldn't remove this. We should just change the top sentence to read something like:

> By default, only the `stdin`, `stdout`, and `stderr` file descriptors are open in the application when started by the runtime.

The last line is **very** important especially when considering `process.terminal = false`.